Sources/Validated.swift

```diff
@@ -1,3 +1,8 @@
+#if swift(>=3.0)
+#else
+    public typealias ErrorProtocol = ErrorType
+#endif
+
```

Comment:
Nice idea! Curios, would it be possible to do this only with an `if` block?

`#if swift(<3.0)`